src/summary/documentation/vital-card-level-two.component.tsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import { match } from "react-router";
+import { performPatientsVitalsSearch } from "./vitals-card.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { formatDate } from "./dimension-helpers";
+import styles from "./vital-card-level-two.css";
+import SummaryCard from "../cards/summary-card.component";
+import { useCurrentPatient } from "@openmrs/esm-api";
+
+export default function VitalsLevelTwo(props: VitalsLevelTwoProps) {
+  const resultsPerPage = 15;
+
+  const [patientVitals, setPatientVitals] = React.useState(null);
+  const [totalPages, setTotalPages] = React.useState(1);
+  const [currentPage, setCurrentPage] = React.useState(1);
+  const [showNextButton, setShowNextButton] = React.useState(false);
+  const [showPreviousButton, setShowPreviousButton] = React.useState(false);
+  const [pagedResults, setPagedResults] = React.useState([]);
```

Comment:
It looks like this is supposed to be the results for the current page, rather than all the results structured into pages. Maybe let's call it `pageResults` or `currentPageResults`?